shared/constants/platform.desktop.js

```diff
@@ -27,196 +23,79 @@ if (__DEV__ && !__STORYBOOK__) {
   console.log(`Run mode: ${runMode}`)
 }
 
-const envedPathLinux = {
```

Comment:
this file is mostly moved around, the output should be the same except for one case (see below)